config.go

```diff
@@ -82,6 +83,23 @@ func (c *Config) InitDefault(log *zap.Logger) {
 	case httpClient:
 	default:
 		c.Client = httpClient
+		// As per https://opentelemetry.io/docs/specs/otel/protocol/exporter/#specify-protocol
```

Comment:
Since this is otel-specific, should this logic for defaults be conditioned on exporter == otlp?